RevenueCatUI/Data/Strings.swift

```diff
@@ -98,6 +103,22 @@ extension Strings: CustomStringConvertible {
 
         case .setting_restored_customer_info:
             return "Setting restored customer info"
+
+        case .executing_external_purchase_logic:
+            return "Will execute custom StoreKit purchase logic provided by your app. " +
+            "No StoreKit purchasing logic will be performed by RevenueCat. " +
+            "You must use `.handlePurchaseAndRestore` on your `PaywallView`."
+
+        case .executing_purchase_logic:
+            return "Will execute purchase logic provided by RevenueCat."
+
+        case .executing_restore_logic:
+            return "Will execute restore purchases logic provided by RevenueCat."
+
+        case .executing_external_restore_logic:
+            return "Will execute custom StoreKit restore purchases logic provided by your app. " +
+            "No StoreKit restore purchases logic will be performed by RevenueCat. " +
```

Comment:
> Will execute custom StoreKit restore purchases logic provided by your app. No StoreKit restore purchases logic will be performed by RevenueCat.

Doesn't quite roll off the tongue, right? It took me a couple of reads to understand this. Maybe we can simplify it a bit? 

how about something like: 
```suggestion
        case .executing_external_restore_logic:
            return "Initiating a restore. Note that the restore logic will be provided by your app and not RevenueCat, since the SDK was configured to blah blah blah...
```

Reply:
But even "Requesting" - requesting to whom? It needs to be clear it's of the app (not StoreKit). 🤔

Reply:
"The user has requested a restore. Note that the restore logic will be provided by your app and not RevenueCat, since the SDK was configured to blah blah blah..."  ?

Reply:
ohh I like that